Error starting OmniSharp: Command failed: cat /etc/*-release

OmniSharp Log:
```
[INFO] Starting OmniSharp at '/home/havoron/Projects/aspnet/cli-samples/HelloMvc'...
[ERROR] Error: Command failed: cat /etc/*-release
cat: /etc/upstream-release: Is a directory
```

There is a directory `/etc/upstream-release/` in Linux Mint 17.3. And the command `cat /etc/*-release` returns:
```
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=17.3
DISTRIB_CODENAME=rosa
DISTRIB_DESCRIPTION="Linux Mint 17.3 Rosa"
NAME="Ubuntu"
VERSION="14.04.4 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.4 LTS"
VERSION_ID="14.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
cat: /etc/upstream-release: Is a directory
```
dotnet --info
```
.NET Command Line Tools (1.0.0-rc2-002411)

Product Information:
 Version:     1.0.0-rc2-002411
 Commit Sha:  e765d572da

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  14.04
 OS Platform: Linux
 RID:         ubuntu.14.04-x64
```
code --version
```
1.0.0
```
OmniSharp: `csharp-1.0.3-rc2.vsix`

--- a/src/utils.ts
+++ b/src/utils.ts
@@ -26,7 +26,7 @@ export function getSupportedPlatform() {
     }
     else if (process.platform === 'linux') {
         // Get the text of /etc/*-release to discover which Linux distribution we're running on.
-        let release = child_process.execSync('cat /etc/*-release').toString().toLowerCase();
+        let release = child_process.execSync('cat /etc/os-release').toString().toLowerCase();
 
         if (release.indexOf('ubuntu') >= 0) {
             return SupportedPlatform.Ubuntu;

